cli/azd/pkg/input/console.go

```diff
@@ -441,6 +444,11 @@ func (c *AskerConsole) Confirm(ctx context.Context, options ConsoleOptions) (boo
 	return response, nil
 }
 
+// wait until the next enter
+func (c *AskerConsole) WaitForEnter() {
+	_ = bufio.NewScanner(c.handles.Stdin).Scan()
```

Comment:
Consider making this error returning and just thread the error up, or failing that, log the error when this fails so we at least have something in our diagnostics log that may help us understand why this failed?

Reply:
added log if error